### ISSUE TITLE
Horrible lag with chromium-based browsers on Linux 

### ISSUE THREAD

**Issue by Joe23232** [brave/brave-browser#8436]
I am using an Arch-based system and any chromium based browser underperforms in everything compared to Windows when using brave and other chromium-based browsers, particularly when scrolling through some websites that have got more content in it.

https://linuxconfig.org/benchmark-your-graphics-card-on-linux This particular website lags a lot on Linux but runs smoothly on Windows.

I have an integrated Intel HD graphics card. I have done two graphics tests using the `phoronix-test-suite` and the first one was very similar (Windows did better by 0.5 fps than Linux) but with the second graphics test Linux excelled by a few frames more than Windows.

I typed `chrome://gpu/` on brave and this has outputted some informaiton. I am not too sure if this is helpful or not but I was wondering if I can  get help to resolve such issues please.

```

Graphics Feature Status
Canvas: Hardware accelerated
Flash: Hardware accelerated
Flash Stage3D: Hardware accelerated
Flash Stage3D Baseline profile: Hardware accelerated
Compositing: Hardware accelerated
Multiple Raster Threads: Enabled
Out-of-process Rasterization: Disabled
OpenGL: Enabled
Hardware Protected Video Decode: Unavailable
Rasterization: Software only. Hardware acceleration disabled
Skia Renderer: Enabled
Video Decode: Unavailable
Viz Display Compositor: Enabled
Vulkan: Disabled
WebGL: Hardware accelerated
WebGL2: Hardware accelerated
Driver Bug Workarounds
adjust_src_dst_region_for_blitframebuffer
clear_uniforms_before_first_program_use
count_all_in_varyings_packing
disable_post_sub_buffers_for_onscreen_surfaces
exit_on_context_lost
msaa_is_slow
rely_on_implicit_sync_for_swap_buffers
scalarize_vec_and_mat_constructor_args
disabled_extension_GL_KHR_blend_equation_advanced
disabled_extension_GL_KHR_blend_equation_advanced_coherent
disabled_extension_GL_MESA_framebuffer_flip_y
Problems Detected
Accelerated video decode is unavailable on Linux: 137247
Disabled Features: accelerated_video_decode
Protected video decoding with swap chain is for Windows and Intel only
Disabled Features: protected_video_decode
Clear uniforms before first program use on all platforms: 124764, 349137
Applied Workarounds: clear_uniforms_before_first_program_use
Mesa drivers in Linux handle varyings without static use incorrectly: 333885
Applied Workarounds: count_all_in_varyings_packing
Disable partial swaps on Mesa drivers (detected with GL_RENDERER): 339493
Applied Workarounds: disable_post_sub_buffers_for_onscreen_surfaces
Always rewrite vec/mat constructors to be consistent: 398694
Applied Workarounds: scalarize_vec_and_mat_constructor_args
On Intel GPUs MSAA performance is not acceptable for GPU rasterization: 527565
Applied Workarounds: msaa_is_slow
Disable partial swaps on Mesa drivers (detected with GL_VERSION): 339493
Applied Workarounds: disable_post_sub_buffers_for_onscreen_surfaces
adjust src/dst region if blitting pixels outside framebuffer on Linux Intel: 664740
Applied Workarounds: adjust_src_dst_region_for_blitframebuffer
Disable KHR_blend_equation_advanced until cc shaders are updated: 661715
Applied Workarounds: disable(GL_KHR_blend_equation_advanced), disable(GL_KHR_blend_equation_advanced_coherent)
Some drivers can't recover after OUT_OF_MEM and context lost: 893177
Applied Workarounds: exit_on_context_lost
Avoid waiting on a egl fence before swapping buffers and rely on implicit sync on Intel GPUs: 938286
Applied Workarounds: rely_on_implicit_sync_for_swap_buffers
Disable GL_MESA_framebuffer_flip_y for desktop GL: 964010
Applied Workarounds: disable(GL_MESA_framebuffer_flip_y)
Version Information
Data exported	2020-02-25T00:33:51.754Z
Chrome version	Chrome/80.0.3987.116
Operating system	Linux 5.5.5-arch1-1
Software rendering list URL	https://chromium.googlesource.com/chromium/src/+/dc00a510e4c2ae25c4d084cc3d946fc782249224/gpu/config/software_rendering_list.json
Driver bug list URL	https://chromium.googlesource.com/chromium/src/+/dc00a510e4c2ae25c4d084cc3d946fc782249224/gpu/config/gpu_driver_bug_list.json
ANGLE commit id	362168eb695d
2D graphics backend	Skia/80 97c9a95908bc8c7a6794259b35f366a3291e2d0f
Command Line	/usr/lib/brave-bin/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --sync-url=https://no-thanks.invalid --enable-features=PasswordImport,SimplifyHttpsIndicator,WebUIDarkMode --disable-features=AutofillServerCommunication,NotificationTriggers,WebXR,AllowPopupsDuringPageUnload,SmsReceiver,AudioServiceOutOfProcess,LookalikeUrlNavigationSuggestionsUI,VideoPlaybackQuality,WebXrGamepadModule --flag-switches-begin --flag-switches-end --disable-webrtc-apm-in-audio-service --disable-sync
Driver Information
Initialization time	360
In-process GPU	false
Passthrough Command Decoder	false
Sandboxed	false
GPU0	VENDOR= 0x8086 [Intel Open Source Technology Center], DEVICE=0x5916 [Mesa DRI Intel(R) HD Graphics 620 (Kaby Lake GT2) ] *ACTIVE*
Optimus	false
AMD switchable	false
Driver vendor	Mesa
Driver version	19.3.4
GPU CUDA compute capability major version	0
Pixel shader version	4.60
Vertex shader version	4.60
Max. MSAA samples	16
Machine model name	
Machine model version	
GL_VENDOR	Intel Open Source Technology Center
GL_RENDERER	Mesa DRI Intel(R) HD Graphics 620 (Kaby Lake GT2)
GL_VERSION	4.6 (Core Profile) Mesa 19.3.4
GL_EXTENSIONS	GL_3DFX_texture_compression_FXT1 GL_AMD_conservative_depth GL_AMD_depth_clamp_separate GL_AMD_draw_buffers_blend GL_AMD_gpu_shader_int64 GL_AMD_multi_draw_indirect GL_AMD_query_buffer_object GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_AMD_shader_trinary_minmax GL_AMD_texture_texture4 GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_APPLE_object_purgeable GL_ARB_ES2_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_compressed_texture_pixel_storage GL_ARB_compute_shader GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_shader GL_ARB_fragment_shader_interlock GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gl_spirv GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_indirect_parameters GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_occlusion_query2 GL_ARB_parallel_shader_compile GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_sprite GL_ARB_polygon_offset_clamp GL_ARB_post_depth_coverage GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_ballot GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_stencil_export GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_spirv_extensions GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map_array GL_ARB_texture_filter_anisotropic GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_buffer_object GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ATI_blend_equation_separate GL_ATI_texture_float GL_EXT_EGL_sync GL_EXT_abgr GL_EXT_blend_equation_separate GL_EXT_demote_to_helper_invocation GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_object GL_EXT_framebuffer_sRGB GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_shader_framebuffer_fetch GL_EXT_shader_framebuffer_fetch_non_coherent GL_EXT_shader_integer_mix GL_EXT_shader_samples_identical GL_EXT_texture_array GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_integer GL_EXT_texture_sRGB GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_decode GL_EXT_texture_shadow_lod GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array_bgra GL_EXT_vertex_attrib_64bit GL_IBM_multimode_draw_arrays GL_INTEL_conservative_rasterization GL_INTEL_performance_query GL_INTEL_shader_atomic_float_minmax GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_context_flush_control GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_KHR_texture_compression_astc_sliced_3d GL_MESA_framebuffer_flip_y GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_signed_rgba GL_NV_compute_shader_derivatives GL_NV_conditional_render GL_NV_depth_clamp GL_NV_fragment_shader_interlock GL_NV_packed_depth_stencil GL_NV_texture_barrier GL_OES_EGL_image GL_S3_s3tc
Disabled Extensions	GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_MESA_framebuffer_flip_y
Disabled WebGL Extensions	
Window system binding vendor	SGI
Window system binding version	1.4
Window system binding extensions	GLX_ARB_create_context GLX_ARB_create_context_no_error GLX_ARB_create_context_profile GLX_ARB_create_context_robustness GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_multisample GLX_EXT_create_context_es_profile GLX_EXT_create_context_es2_profile GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_import_context GLX_EXT_libglvnd GLX_EXT_no_config_context GLX_EXT_texture_from_pixmap GLX_EXT_visual_info GLX_EXT_visual_rating GLX_MESA_copy_sub_buffer GLX_OML_swap_method GLX_SGI_make_current_read GLX_SGI_swap_control GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_INTEL_swap_event
Window manager	KWin
XDG_CURRENT_DESKTOP	KDE
Compositing manager	Yes
Direct rendering version	DRI3
Reset notification strategy	0x8252
GPU process crash count	0
System visual ID	32
RGBA visual ID	408
gfx::BufferFormats supported for allocation and texturing	R_8: not supported, R_16: not supported, RG_88: not supported, BGR_565: not supported, RGBA_4444: not supported, RGBX_8888: not supported, RGBA_8888: not supported, BGRX_8888: not supported, BGRX_1010102: not supported, RGBX_1010102: not supported, BGRA_8888: not supported, RGBA_F16: not supported, YVU_420: not supported, YUV_420_BIPLANAR: not supported, P010: not supported
Compositor Information
Tile Update Mode	One-copy
Partial Raster	Enabled
GpuMemoryBuffers Status
R_8	Software only
R_16	Software only
RG_88	Software only
BGR_565	Software only
RGBA_4444	Software only
RGBX_8888	Software only
RGBA_8888	Software only
BGRX_8888	Software only
BGRX_1010102	Software only
RGBX_1010102	Software only
BGRA_8888	Software only
RGBA_F16	Software only
YVU_420	Software only
YUV_420_BIPLANAR	Software only
P010	Software only
Display(s) Information
Info	Display[1657942003464773] bounds=[0,0 1920x1080], workarea=[0,0 1920x1044], scale=1, rotation=0, panel_rotation=0 external.
Color space information	{primaries:INVALID, transfer:INVALID, matrix:INVALID, range:INVALID}
SDR white level in nits	80
Bits per color component	8
Bits per pixel	24
Refresh Rate in Hz	60
Video Acceleration Information
Vulkan Information
Log Messages
[1319:1319:0225/110433.052231:ERROR:vaapi_wrapper.cc(417)] : vaInitialize failed: unknown libva error
[1319:1319:0225/110433.335597:ERROR:sandbox_linux.cc(374)] : InitializeSandbox() called with multiple threads in process gpu-process.
[1319:1319:0225/113351.344323:WARNING:x11_util.cc(1443)] : X error received: serial 250326, error_code 170 (GLXBadWindow), request_code 152, minor_code 32 (X_GLXDestroyWindow)
[1319:1319:0225/113351.344395:WARNING:x11_util.cc(1443)] : X error received: serial 250330, error_code 3 (BadWindow (invalid Window parameter)), request_code 4, minor_code 0 (X_DestroyWindow)
```

Is there something i need to install or something to help accelerate the performance of brave?

If the log is hard to read, I have made a document which makes it easier to read it as all the formatting of the document (such as bold text etc) has been copied over to the document.

https://drive.google.com/open?id=1pT7XBCty5Ej1D3LAZnsFF9CdIq3ytEkg

---

When I typed this out in the terminal `brave --enable-gpu-rasterization --ignore-gpu-blacklist --disable-gpu-driver-workarounds` and I tested out this website https://linuxconfig.org/benchmark-your-graphics-card-on-linux.

It does seem to be smoother but still lags. And for some reason I am getting errors in the console when I ran the command mentioned above.

```
[4374:4374:0225/150641.768461:ERROR:vaapi_wrapper.cc(417)] vaInitialize failed: unknown libva error
[4374:4374:0225/150641.828119:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[4350:4366:0225/150641.936944:ERROR:rewards_service_impl.cc(206)] Failed to read file: /home/user1/.config/BraveSoftware/Brave-Browser/Default/ledger_state
[4382:4382:0225/150642.015997:ERROR:ledger_impl.cc(102)] Failed to initialize wallet
[4432:1:0225/150642.477159:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.482840:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.555189:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.556842:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.562601:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.569936:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.573624:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.581982:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.583865:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4432:1:0225/150642.586019:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4499:1:0225/150644.002756:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4499:1:0225/150644.007219:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4350:4350:0225/150655.061023:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Cannot access contents of url "chrome://gpu/". Extension manifest must request permission to access this host.", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/_generated_background_page.html (0)
[4350:4350:0225/150655.061087:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Cannot access contents of url "chrome://newtab/". Extension manifest must request permission to access this host.", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/_generated_background_page.html (0)
[4350:4350:0225/150656.624712:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Cannot access contents of url "chrome://media-internals/". Extension manifest must request permission to access this host.", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/_generated_background_page.html (0)
[4644:1:0225/150658.675364:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.675885:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.676499:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.677151:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.678485:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.679185:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.680367:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.684527:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.685787:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4644:1:0225/150658.686406:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.
[4350:4350:0225/150702.394885:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Cannot access contents of url "chrome://gpu/". Extension manifest must request permission to access this host.", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/_generated_background_page.html (0)
[4350:4350:0225/150702.416069:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Cannot access contents of url "chrome://settings/". Extension manifest must request permission to access this host.", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/_generated_background_page.html (0)
[4350:4350:0225/150702.651868:ERROR:CONSOLE(4)] "Polymer is already defined.", source: chrome://settings/crisper.js (4)
```

Is there something else that needs to be enabled/disabled in order to fully fix the lag issues?

**Comment by Joe23232** [comment]
Chromium-based browsers are all unnecessarily taking up too much CPU processing power. I pressed `shift`+`esc` on both Windows and Linux within the browsers and I found that Brave was taking up way too much CPU processing power compared to Windows. So I compared two websites [Benchmark Your Graphics Card On Linux](https://linuxconfig.org/benchmark-your-graphics-card-on-linux) and [WhatsApp Web](https://web.whatsapp.com/)

# CPU use for Brave for Windows

## Benchmark Your Graphics Card On Linux
Idle:		1-4
Scrolling:	30-40

## WhatsApp Web
Idle:		0
Scrolling:	30-60

## GPU Process (for Benchmark Your Graphics Card On Linux website)
Idle:		0-3
Scrolling:	18-48

---

# CPU use for Brave for EndeavourOS

## Benchmark Your Graphics Card On Linux
Idle:		17
Scrolling:	140-160

## WhatsApp Web
Idle:		0
Scrolling:	153

## GPU Process (for Benchmark Your Graphics Card On Linux website)
Idle:		2-5
Scrolling:	110

---

So I am not too sure why on Linux Chromium-based browsers are taking up more CPU processing power compared to Windows, if anyone knows what is going on I would love to know. Thanks :)

**Comment by NumDeP** [comment]
Hi @Joe23232 just out of curiosity have you ever had issues like this https://github.com/brave/brave-browser/issues/6712? 

I ask because I'm on `Version 1.5.123 Chromium: 80.0.3987.163 (Official Build) (64-bit) (openSuse)` and I've sort of noticed this lag more often though not exactly like your issue.

For the team if it's any help - 

```
Graphics Feature Status
Canvas: Software only, hardware acceleration unavailable
Flash: Software only. Hardware acceleration disabled
Flash Stage3D: Software only. Hardware acceleration disabled
Flash Stage3D Baseline profile: Software only. Hardware acceleration disabled
Compositing: Software only. Hardware acceleration disabled
Multiple Raster Threads: Enabled
Out-of-process Rasterization: Disabled
OpenGL: Disabled
Hardware Protected Video Decode: Disabled
Rasterization: Software only. Hardware acceleration disabled
Skia Renderer: Enabled
Video Decode: Software only. Hardware acceleration disabled
Viz Display Compositor: Enabled
Vulkan: Disabled
WebGL: Software only, hardware acceleration unavailable
WebGL2: Software only, hardware acceleration unavailable
Problems Detected
Gpu compositing has been disabled, either via blacklist, about:flags or the command line. The browser will fall back to software compositing and hardware acceleration will be unavailable.
Disabled Features: gpu_compositing
```

**Comment by bsclifton** [comment]
This issue hasn't had any comments in 4 years - I don't think there is anything actionable we can do

If there is something specific we can follow up on, let's create a new issue please and include those details. Thanks!

**Comment by Joe23232** [comment]
It turns out back then chromium didn't support hardware acceleration unless forced via chromium flags.
